src/test/java/io/anserini/index/IndexReaderUtilsTest.java

```diff
@@ -79,6 +81,36 @@ public void testTermCounts() throws Exception {
     assertEquals(Long.valueOf(2), termCountMap.get("docFreq"));
   }
 
+  @Test
+  public void testIterateThroughTerms() throws Exception {
+    Directory dir = FSDirectory.open(tempDir1);
+    IndexReader reader = DirectoryReader.open(dir);
+
+    TermsEnum iter = IndexReaderUtils.getTermIterator(reader);
+    BytesRef bytesRef = iter.next();
+    while (bytesRef != null) {
+      // This is the current term in the dictionary.
+      String token = bytesRef.utf8ToString();
+      Term term = new Term("contents", token);
+
+      // How to dump out positional info as well, from test case at:
```

Comment:
I think you can remove this part about the positions... but I think you want to add some actual `assertEquals`?